web3/utils/gas_price_engines.py

```diff
@@ -0,0 +1,4 @@
+
+
+def default_gas_price_engine(web3, transaction_params):  # TODO: call eth.gasPrice
```

Comment:
What do you think about moving this to `web3.gas_engine.rpc` and renaming it to `rpc_gas_price_engine`.

Reply:
I'm not sure I understand what the `rpc` part of the package and module names really mean?  I'm happy to give this it's own package though

On the subject of naming, how do you feel about an alternative name "gas pricing strategy"?

Reply:
I'm not in love with the name `rpc` but the idea was that this engine get's the price via the JSON-RPC connection.  `default` is a tempting name but I was hoping to come up with a name that was more descriptive of what it actually does.  Maybe `rpc` is worse.

I'm good going forward with `default` as the namespace.

> On the subject of naming, how do you feel about an alternative name "gas pricing strategy"?

I like it.  Make it so!

Reply:
Oh i see what you mean. I've changed it to `rpc_gas_pricing_strategy`. I'll have a think if there's a better name that talks more accurately but this should do for now